backend/models/postgis/project.py

```diff
@@ -1139,16 +1139,19 @@ def calculate_tasks_percent(
         target, total_tasks, tasks_mapped, tasks_validated, tasks_bad_imagery
     ):
         """ Calculates percentages of contributions """
-        if target == "mapped":
-            return int(
-                (tasks_mapped + tasks_validated)
-                / (total_tasks - tasks_bad_imagery)
-                * 100
-            )
-        elif target == "validated":
-            return int(tasks_validated / (total_tasks - tasks_bad_imagery) * 100)
-        elif target == "bad_imagery":
-            return int((tasks_bad_imagery / total_tasks) * 100)
+        try:
```

Comment:
could you just check if total_tasks == tasks_bad_imagery instead of using an exception for control flow?
I also think that we need to catch this case in the [project explore result](https://github.com/DigitalGlobe/tasking-manager/blob/master/backend/services/project_search_service.py#L101-L102)